packages/perseus/src/util/graphie.ts

```diff
@@ -4,6 +4,7 @@ import {
     vector as kvector,
     number as knumber,
 } from "@khanacademy/kmath";
+import {entries} from "@khanacademy/wonder-stuff-core";
```

Comment:
Now that we're using TS, I don't think we need to use this anymore. 

Webapp is just in the process of dropping the requirement to use `entries` and allow simply using `Object.entries()` directly. 

https://github.com/Khan/webapp/pull/19488

Reply:
It looks like `Object.entries` still doesn't have a very precise return type, so I'd prefer to keep using `entries`. I left a comment on the webapp PR.